### PR TITLE
fix: build rockpi4 metal image as part of CI build

### PR DESCRIPTION
**Pull request by Unix4ever**
Added rockpi_4 to the list of sbcs targets in the Makefile.

Signed-off-by: Artem Chernyshev <artem.0xD2@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -174,7 +174,7 @@ sbc-%: ## Builds the specified SBC image. Valid options are rpi_4, rock64, banan
 	@docker pull $(REGISTRY_AND_USERNAME)/installer:$(TAG)
 	@docker run --rm -v /dev:/dev --privileged $(REGISTRY_AND_USERNAME)/installer:$(TAG) image --platform metal --board $* --tar-to-stdout | tar xz -C $(ARTIFACTS)
 
-sbcs: sbc-rpi_4 sbc-rock64 sbc-bananapi_m64 sbc-libretech_all_h3_cc_h5 ## Builds all known SBC images (Raspberry Pi 4 Model B, Rock64, Banana Pi M64, and Libre Computer Board ALL-H3-CC).
+sbcs: sbc-rpi_4 sbc-rock64 sbc-bananapi_m64 sbc-libretech_all_h3_cc_h5 rockpi_4 ## Builds all known SBC images (Raspberry Pi 4 Model B, Rock64, Banana Pi M64, and Libre Computer Board ALL-H3-CC).
```
Comment (AlekSi) [inline Makefile:177]: I _think_ it should be `sbc-rockpi_4`. And maybe update comment too?

Comment (Unix4ever) [reply Makefile]: _nod_

Comment (AlekSi) [reply Makefile]: I don't think comment was updated :)

> Builds all known SBC images (Raspberry Pi 4 Model B, Rock64, Banana Pi M64, and Libre Computer Board ALL-H3-CC)

ROCK Pi 4 is not included in the list

Comment (Unix4ever) [reply Makefile]: fixed :facepalm:

**Review by smira — APPROVED** [review]

**Pull request merged**